### PR TITLE
docs(graphql): remove reference of non-existent gms.graphql

### PR DESCRIPTION
**Pull request by mayurinehate**
Issue reported in this slack thread - https://datahubspace.slack.com/archives/C029A3M079U/p1675416618086009

## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-graphql-core/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ permitting reads and writes against the entities and aspects on the Metadata Gra
 
 Contained within this module are 
 
-1. **GMS Schema**: A GQL schema that based on GMS models, located under `resources/gms.graphql`.
```
Comment (anshbansal) [inline datahub-graphql-core/README.md:11]: There is no folder `datahub-graphql-core/resources/` either. So what is this new location pointing to?

Comment (mayurinehate) [reply datahub-graphql-core/README.md:11]: `src/main/resources`

Comment (anshbansal) [reply datahub-graphql-core/README.md:11]: This doc will go into public docs website. So we will need a URL that people looking at public docs can understand.

Comment (mayurinehate) [reply datahub-graphql-core/README.md:11]: updated to add link.

**Review by anshbansal — APPROVED** [review]

**Pull request merged**